frontend/package.json

```diff
@@ -46,6 +46,7 @@
     "test-cypress-shipwright": "cd packages/shipwright-plugin/integration-tests && yarn run test-cypress",
     "test-cypress-shipwright-headless": "cd packages/shipwright-plugin/integration-tests && yarn run test-cypress-headless",
     "test-cypress-shipwright-nightly": "cd packages/shipwright-plugin/integration-tests && yarn run test-cypress-headless-all",
+    "test-cypress-webterminal": "cd packages/webterminal-plugin/integration-tests && yarn run test-cypress",
```

Comment:
All other plugins has also a -headless and a -nighly variant. Let us do it similar for this plugin.

You should use them then in the `test-cypress.sh` script as well (in the git root folder)

Reply:
Added -headless and -nightly.. we don't have e2e/feature file. So added `--spec \"features/*/*.feature\"` for both test-cypress-headless and test-cypress-headless-all

Reply:
:+1: